helpers/permissions.ts

```diff
@@ -34,6 +34,13 @@ export const getRedstoneOraclePermissions = (redstoneOracle: string): string[][]
   ];
 };
 
+export const getRedstoneOraclePermissionsRevokedPreviously = (redstoneOracle: string, guardian: string): string[][] => {
```

Comment:
I think I would add this function to the file deploy/010-redstone-and-vault-permission.ts, because it is specific for that script and I don't think we'll reuse it on other scripts

Reply:
[984744e](https://github.com/VenusProtocol/governance-contracts/pull/102/commits/984744e0affdf2e49512ce033acf2bbe7440b2dd)